src/setup-ros2.ts

```diff
@@ -2,179 +2,205 @@ import * as core from "@actions/core";
+
+	await installPython3Dependencies();
+
+	// Initializes rosdep
+	await exec.exec("sudo", ["rosdep", "init"]);
 }
 
 async function runOsX() {
-  await runBrew([
-    "asio",
-    "assimp",
-    "console_bridge",
-    "cppcheck",
-    "eigen",
-    "freetype",
-    "log4cxx",
-    "opencv",
-    "openssl",
-    "pcre",
-    "poco",
-    "python3",
-    "qt",
-    "tinyxml",
-    "tinyxml2",
-    "wget"
-  ]);
-  await exec.exec(
-    "sudo", [
-      "bash",
-      "-c",
-      "echo \"export OPENSSL_ROOT_DIR=$(brew --prefix openssl)\" >> ~/.bashrc"
-  ]);
-  await exec.exec(
-    "sudo", [
-      "bash",
-      "-c",
-      "echo \"export CMAKE_PREFIX_PATH=$CMAKE_PREFIX_PATH:/usr/local/opt/qt\" >> ~/.bashrc"
-  ]);
-  await exec.exec(
-    "sudo", [
-      "bash",
-      "-c",
-      "echo \"export PATH=$PATH:/usr/local/opt/qt/bin\" >> ~/.bashrc"
-  ]);
-  await installPython3Dependencies();
-
-  // While rosdep and vcs are available as a Debian package on Ubuntu, they need
-  // to be installed through pip on OS X.
-  await exec.exec(
-   "sudo", ["pip3", "install", "--upgrade", "rosdep", "vcstool"])
-  // Initializes rosdep
-  await exec.exec("sudo", ["rosdep", "init"])
+	await runBrew([
+		"asio",
+		"assimp",
+		"console_bridge",
+		"cppcheck",
+		"eigen",
+		"freetype",
+		"log4cxx",
+		"opencv",
+		"openssl",
+		"pcre",
+		"poco",
+		"python3",
+		"qt",
+		"tinyxml",
+		"tinyxml2",
+		"wget"
+	]);
+	await exec.exec("sudo", [
+		"bash",
+		"-c",
+		'echo "export OPENSSL_ROOT_DIR=$(brew --prefix openssl)" >> ~/.bashrc'
+	]);
+	await exec.exec("sudo", [
+		"bash",
+		"-c",
+		'echo "export CMAKE_PREFIX_PATH=$CMAKE_PREFIX_PATH:/usr/local/opt/qt" >> ~/.bashrc'
+	]);
+	await exec.exec("sudo", [
+		"bash",
+		"-c",
+		'echo "export PATH=$PATH:/usr/local/opt/qt/bin" >> ~/.bashrc'
+	]);
+	await installPython3Dependencies();
+
+	// While rosdep and vcs are available as a Debian package on Ubuntu, they need
+	// to be installed through pip on OS X.
+	await exec.exec("sudo", [
+		"pip3",
+		"install",
+		"--upgrade",
+		"rosdep",
+		"vcstool"
+	]);
+	// Initializes rosdep
+	await exec.exec("sudo", ["rosdep", "init"]);
 }
 
 async function runWindows() {
-  core.setFailed("Windows is not yet supported");
+	core.setFailed("Windows is not yet supported");
 }
 
 async function run() {
-  try {
-    const platform = process.platform;
-    if (platform === "darwin") {
-      await runOsX();
-    } else if (platform === "win32") {
-      await runWindows();
-    } else if (platform === "linux") {
-      await runLinux();
-    } else {
-      core.setFailed(`unsupport platform ${platform}`);
-    }
-  } catch (error) {
-    core.setFailed(error.message);
-  }
+	try {
+		const platform = process.platform;
+		if (platform === "darwin") {
+			await runOsX();
+		} else if (platform === "win32") {
+			await runWindows();
+		} else if (platform === "linux") {
+			await runLinux();
+		} else {
+			core.setFailed(`unsupport platform ${platform}`);
```

Comment:
Unsupported*

Reply:
done - but this is due to linting, not a new change